app/src/main/res/values/strings.xml

```diff
@@ -110,4 +110,7 @@
     <string name="title_average">Average</string>
     <string name="title_customer_old">Old</string>
     <string name="title_customer_new">New</string>
+    <string name="title_stylist_phone">Stylist Phone</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_stylist_phone` appears to be unused
 Unused resources make applications larger and slow down builds.